packages/ui/src/utils/truncate-label.tsx

```diff
@@ -0,0 +1,5 @@
+export const doTruncateLabel = (label: string | undefined): string | undefined => {
+  if (label === undefined) return;
+
+  return label.length > 15 ? label.substring(0, 12) + '...' : label;
```

Comment:
Could you please check on the patternfly side if something like this is already provided?